api/tx_args.go

```diff
@@ -53,27 +101,27 @@ type NewTxArgs interface {
 
 // SendTxArgs represents the arguments to submit a new transaction into the transaction pool.
 type SendTxArgs struct {
-	TypeInt  *types.TxType   `json:"typeInt"`
-	From     common.Address  `json:"from"`
-	To       *common.Address `json:"to"`
-	Gas      *hexutil.Uint64 `json:"gas"`
-	GasPrice *hexutil.Big    `json:"gasPrice"`
-	Value    *hexutil.Big    `json:"value"`
-	Nonce    *hexutil.Uint64 `json:"nonce"`
+	TypeInt      *types.TxType   `json:"typeInt"`
+	From         common.Address  `json:"from"`
+	Recipient    *common.Address `json:"to"`
+	GasLimit     *hexutil.Uint64 `json:"gas"`
+	Price        *hexutil.Big    `json:"gasPrice"`
+	Amount       *hexutil.Big    `json:"value"`
+	AccountNonce *hexutil.Uint64 `json:"nonce"`
 	// We accept "data" and "input" for backwards-compatibility reasons. "input" is the
 	// newer name and should be preferred by clients.
-	Data  *hexutil.Bytes `json:"data"`
-	Input *hexutil.Bytes `json:"input"`
+	Data    *hexutil.Bytes `json:"data"`
+	Payload *hexutil.Bytes `json:"input"`
 
 	CodeFormat    *params.CodeFormat `json:"codeFormat"`
 	HumanReadable *bool              `json:"humanReadable"`
 
-	AccountKey *hexutil.Bytes `json:"Key"`
+	Key *hexutil.Bytes `json:"key"`
 
 	FeePayer *common.Address `json:"feePayer"`
 	FeeRatio *types.FeeRatio `json:"feeRatio"`
 
-	Signatures types.TxSignaturesJSON `json:"Signatures"`
+	TxSignatures types.TxSignaturesJSON `json:"signatures"`
```

Comment:
@jimni1222 @nogang Caver can support API with `key`, `signatures` json name?

Reply:
For now in caver-js, sendTransaction works only for legacy transaction.
If user wants to send signed transaction, caver-js will send RLP-encoded transaction string with klay_sendRawTransaction.
Before release this feature, caver needs to support this :)

Reply:
`key`, `signatures` fields were introduced a few days ago. Support of caver is not considered yet.

Reply:
Thanks. :)